src/Controller/HomeController.php

```diff
@@ -4,18 +4,16 @@
 
 use App\Contributors\Service\Repository as ContributorsRepository;
 use App\Contributors\Model\Contributor;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Response;
 
 class HomeController extends Controller
 {
     /**
-     * @Route("/", name="homepage")
      * @param ContributorsRepository $contributorsRepository
      * @return Response
      */
-    public function index(ContributorsRepository $contributorsRepository): Response
+    public function indexAction(ContributorsRepository $contributorsRepository): Response
```

Comment:
We should do the opposite: remove "Action" from every action name

Reply:
I know! For now i have allined with other controllers. In another PR i do a refactoring all controller without `Action`! Ok?